### PR TITLE
Header for file title and `document-loaded` event.

### LINKED ISSUE #6: Add a header to the viewer
For zooming, switching pages and showing title of document.

### PR DESCRIPTION
**Pull request by Peppe**
Fixes #6

### CODE REVIEW

File: src/vaadin-pdf-viewer.html [hunk 1]
```diff
@@ -81,10 +83,45 @@
...
+             * The pdf metadata title
+             */
+            _pdfTitle: String
+
           };
         }
 
+        static get observers() {
+          return [
+            'setTitle(_pdfTitle, _filename)'
```
Comment (yuriy-fix) [inline src/vaadin-pdf-viewer.html]: Nit: observer method is better to be private.

File: src/vaadin-pdf-viewer.html [hunk 2]
```diff
@@ -81,10 +83,45 @@
...
+          ];
+        }
+
+        setTitle(_pdfTitle, _filename) {
```
Comment (yuriy-fix) [inline src/vaadin-pdf-viewer.html]: No need in specifying parameters with underscore (can be just pdfTitle and filename)

File: src/vaadin-pdf-viewer.html [hunk 3]
```diff
@@ -81,10 +83,45 @@
             /**
              * A represenentation of a document that has been read in. 
              */
-            _document: Object
+            _document: Object,
```
Comment (yuriy-fix) [inline src/vaadin-pdf-viewer.html]: Probably it's a good idea to make all of those properties private, not protected.
https://polymer-library.polymer-project.org/3.0/docs/devguide/properties#private-and-protected-properties

File: demo/pdf-viewer-basic-demos.html [hunk 4]
```diff
@@ -27,6 +27,24 @@ <h3>No File Specified</h3>
...
+    <vaadin-demo-snippet id="pdf-viewer-basic-demos-document-loaded" when-defined="vaadin-pdf-viewer">
+      <template preserve-content>
+        <vaadin-pdf-viewer src="pdf-files/example-invoice.pdf"></vaadin-pdf-viewer>
+        <div id="status">Loading...</div>
+        <script>
+          window.addDemoReadyListener('#pdf-viewer-basic-demos-document-loaded', function (document) {
+            var start = new Date().getTime();
+            const viewer = document.querySelector('vaadin-pdf-viewer');
+            viewer.addEventListener('document-loaded', function (e) {
+
```
Comment (yuriy-fix) [inline demo/pdf-viewer-basic-demos.html]: Nit: remove empty line.

File: src/vaadin-pdf-viewer.html (hunk 1 above)
Comment (Peppe) [reply src/vaadin-pdf-viewer.html]: I guess by private you mean that there is an underscore before the name?

Comment (Peppe) [reply src/vaadin-pdf-viewer.html]: Done. I added an underscore to the name. I'm not sure if this should have the double underscore as well, or is it just a properties thing?

File: src/vaadin-pdf-viewer.html (hunk 2 above)
Comment (Peppe) [reply src/vaadin-pdf-viewer.html]: Done.

File: src/vaadin-pdf-viewer.html (hunk 3 above)
Comment (Peppe) [reply src/vaadin-pdf-viewer.html]: Done. I made them __viewer, __document, __title, __filename and __pdfTitle

File: demo/pdf-viewer-basic-demos.html (hunk 4 above)
Comment (Peppe) [reply demo/pdf-viewer-basic-demos.html]: Done.

**Pull request merged**